src/app/6-injectable/my-injectable3/my-injectable3.component.html

```diff
@@ -0,0 +1 @@
+{{this.myComponentVar}}
```

Comment:
No es buena práctica usar this en el HTML, con que declares {{myComponentVar}} es suficiente

Reply:
otro comentario resuelto sin cambios en el código :/